src/Symfony/Component/Routing/Tests/Fixtures/dumper/static_url_matcher5.php

```diff
@@ -0,0 +1,199 @@
+<?php
```

Comment:
I appreciate the efforts to generate a somewhat readable matcher files, but looking at the generated files (like this one) I wonder if it's readable at all with the route definition spanning so many lines.

Reply:
should be much better now

Reply:
It's great now. Thank you!